### ISSUE TITLE
Not working in  Slicer 4.10 stable release nor 4.11 nightly build with Windows 7. 

### ISSUE THREAD

**Issue by edwinlee2006** [Radiomics/SlicerRadiomics#51]
tried SlicerRadiomics in 3D slicer. Successfully installed. But keep getting the following message. Try it in both 4.10 stable release and 4.11 nightly build. Also tried pip to install pywavelets. The python console shows it has been installed. 

==================================================================

_Requirement already satisfied: pywavelets in c:\users\usersname\appdata\roaming\na-mic\extensions-27501\slicerradiomics\lib\site-packages (1.0.0)
Requirement already satisfied: numpy>=1.9.1 in c:\program files\slicer 4.10.0\lib\python\lib\site-packages\numpy-1.13.1-py2.7-win-amd64.egg (from pywavelets) (1.13.1)
0_

===============================================================
**RadiomicsCLI standard error:

Traceback (most recent call last):
  File "C:/Users/usersname/AppData/Roaming/NA-MIC/Extensions-27501/SlicerRadiomics/lib/Slicer-4.10/cli-modules/SlicerRadiomicsCLI.py", line 6, in 
    from radiomics.scripts import parse_args
  File "C:\Users\usersname\AppData\Roaming\NA-MIC\Extensions-27501\SlicerRadiomics\Lib\site-packages\radiomics\__init__.py", line 15, in 
    from . import imageoperations
  File "C:\Users\usersname\AppData\Roaming\NA-MIC\Extensions-27501\SlicerRadiomics\Lib\site-packages\radiomics\imageoperations.py", line 6, in 
    import pywt
  File "C:\Users\usersname\AppData\Roaming\NA-MIC\Extensions-27501\SlicerRadiomics\Lib\site-packages\pywt\__init__.py", line 16, in 
    from ._extensions._pywt import *
ImportError: DLL load failed: The specified module could not be found.**

**Comment by pieper** [comment]
Looks like this is a return of #50 or at least related.

**Comment by fedorov** [comment]
@edwinlee2006 what version of Windows do you have? do you have all updates installed?

Slicer dashboard does not show the log (http://slicer.cdash.org/viewTest.php?onlyfailed&buildid=1448928), and I won't have an opportunity to test on Windows until next week.

**Comment by edwinlee2006** [comment]
> @edwinlee2006 what version of Windows do you have? do you have all updates installed?
> 
> Slicer dashboard does not show the log (http://slicer.cdash.org/viewTest.php?onlyfailed&buildid=1448928), and I won't have an opportunity to test on Windows until next week.

It is Windows 7 Pro. I uninstalled older version of the 3D slicer, and then downloaded and installed the 4.10 stable release and 4.11 nightly build two days ago. After that, I followed the SlicerRadiomics' instruction and installed the Radiomics with the extension management tool in the 3D Slicer. The error messages were the same in both 4.10 stable and 4.11 nightly build. I then checked the PyWavelets with pip. Pip shows the PyWavelets is installed (ver. 1.0.0).

**Comment by JoostJM** [comment]
Hi @edwinlee2006,

I ran into this error too. I've been investigating it and I think I found the error. Recently I fixed the version for PyWavelets toi <= 1.0.0, because version 1.0.1 does not seem to compile on Slicer Build factory. I changed this in the PyRadiomics requirements.txt (correct) and in the cmake build file, where it is specified as the argument for `--no-binary` (bug!). 

This last change resulted in the `--no-binary` argument having no effect, meaning that on the windows build, it simply installed the wheel, which does not work as it is compiled using a different compiler (therefore resulting in the DLL load fail you're seeing). I fixed this by reverting the change in SlicerRadiomics today. If you download the slicer nightly tomorrow, it should be fixed.
